src/compiler/utilitiesPublic.ts

```diff
@@ -2006,8 +2006,6 @@ export function isLiteralTypeLiteral(node: Node): node is NullLiteral | BooleanL
 /**
  * Determines whether a node is an expression based only on its kind.
  * Use `isExpressionNode` if not in transforms.

```

Comment:
@rbuckton This is why I exported `isExpressionNode` we can drop this comment if it seems confusing and we don't want `isExpressionNode` out there.

Reply:
That comment was for us, I would just remove it for the public API.